### DISCUSSION TITLE
Presentation

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#34] [General]
This is te presentation. In the second page im going to change for the image of the paper tomorrow in the institute. 
Please make comments and posibles changes. 
I send in the two formats, presentation of power point and PDF

[contenido seminario 1.pptx](https://github.com/will-henney/globule-seminario/files/11648541/contenido.seminario.1.pptx)

[contenido seminario 1.pdf](https://github.com/will-henney/globule-seminario/files/11648550/contenido.seminario.1.pdf)

**Comment by will-henney** [comment]
Gracias Roberto

En seguida lo reviso

Saludos

  Dr William Henney, Instituto de Radioastronomía y Astrofísica,
  Universidad Nacional Autónoma de México, Campus Morelia

**Comment by will-henney** [comment]
This is mostly fine. I do have some comments on the figure on page 6

![CleanShot 2023-06-05 at 13 36 31@2x](https://github.com/will-henney/globule-seminario/assets/161264/7cc90304-84f8-420d-8bc3-29edfaa555bd)

I think you need to move the labels for rho_0 and rho_1

**Comment by will-henney** [comment]
https://ui.adsabs.harvard.edu/abs/2018MNRAS.477.2431T/abstract

This is link to Tarango-Yong & Henney (2018)

**Comment by RobeReyes** [comment]
The presentation with the changes.

[contenido.seminario.1.pptx](https://github.com/will-henney/globule-seminario/files/11658449/contenido.seminario.1.pptx)

**Comment by will-henney** [comment]
This is now fine, except:
- "cúmulos globulares" is not right
- "emission measure" (not "meassure")

**Comment by RobeReyes** [comment]
[Seminario1.pptx](https://github.com/will-henney/globule-seminario/files/11661086/contenido.seminario.1.pptx)
